src/main/java/com/sprint/mission/discodeit/entity/Message.java

```diff
@@ -0,0 +1,73 @@
+package com.sprint.mission.discodeit.entity;
+
+import com.sprint.mission.discodeit.service.jcf.JCF_user;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.UUID;
+
+public class Message {
+    private UUID id;
+    private Long createdAt;
+    private Long updatedAt;
+    private JCF_user jcfUser = new JCF_user();
+    private String userName;
+    private UUID userId;
+
+    private UUID channelId;
+
+    private String content;
+
+    public Message(UUID userId, String content, UUID channelId, String userName){
+        this.id = UUID.randomUUID();
+        this.createdAt = System.currentTimeMillis();
+        this.userId = userId;
+        this.content = content;
+        this.channelId = channelId;
+        this.userName = userName;
+    }
+
+    public UUID getId() {
+        return id;
+    }
+
+    public Long getCreatedAt() {
+        return createdAt;
+    }
+
+    public UUID getUserId() {
+        return userId;
+    }
+
+    public boolean isUserEqual(UUID getUser) {
```

Comment:
파라미터 명은 userId가 좀 더 적절할 것 같아요.  
```suggestion
    public boolean isUserEqual(UUID userId) {
```